### PR TITLE
Adjust to HTTP configuration changes in the server for 3.0

### LINKED ISSUE #369: Adapt documentation to new HTTP settings in Graylog 3.0.0
Refs Graylog2/graylog2-server#3840

### PR DESCRIPTION
**Pull request by bernd**
Fixes #369

### CODE REVIEW

File: pages/configuration/https.rst [hunk 1]
```diff
@@ -6,10 +6,7 @@
 Using HTTPS
 ***********
 
-We highly recommend securing your Graylog installation using SSL/TLS to make sure that no sensitive data is sent over the wire in plain text. To make this work, you need to do two things:
-
-  - Enable TLS for the Graylog REST API (``rest_enable_tls``)
-  - Enable TLS for the web interface endpoint (``web_enable_tls``)
+We highly recommend securing your Graylog installation using SSL/TLS to make sure that no sensitive data is sent over the wire in plain text. To make this work, you need to enable the ``http_enable_tls`` setting in your Graylog server configuration.
 
 You also need to make sure that you have proper certificates in place, which are valid and trusted by the clients. Not enabling TLS for either one of them will result in a browser error about mixed content and the web interface will cease to work.
```
Comment (jalogisch) [inline pages/configuration/https.rst]: This line makes no sense as all is one

File: pages/configuration/rest_api.rst [hunk 2]
```diff
@@ -6,7 +6,7 @@ Graylog REST API
 
 The functionality Graylog REST API is very comprehensive; even the Graylog web interface is exclusively using Graylog REST API to interact with the Graylog cluster.
 
-To connect to the Graylog REST API with a web browser, just add ``api-browser`` to your current ``rest_listen_uri`` setting or use the **API browser** button on the nodes overview page (*System / Nodes* in the web interface).
+To connect to the Graylog REST API with a web browser, just add ``api-browser`` to your current ``http_publish_uri`` setting or use the **API browser** button on the nodes overview page (*System / Nodes* in the web interface).
```
Comment (jalogisch) [inline pages/configuration/rest_api.rst]: In this case it looks false - it would be ``http_publish_uri``/api/ where the `api-browser` needs to be added.

**Review by jalogisch — CHANGES_REQUESTED** [review]
I like that you add the example for hostname.tld/graylog with NGINX!

Just two little comments.

File: pages/configuration/https.rst (hunk 1 above)
Comment (bernd) [reply pages/configuration/https.rst]: Good catch. :+1:

File: pages/configuration/rest_api.rst (hunk 2 above)
Comment (bernd) [reply pages/configuration/rest_api.rst]: Aye :+1:

**Review by jalogisch — APPROVED** [review]
LGTM

**Comment by jalogisch** [comment]
maybe @joschi can check the settings parts if something is wrong or missing.

**Pull request merged**